app/actions/remote/team.ts

```diff
@@ -98,6 +98,46 @@ export async function addUserToTeam(serverUrl: string, teamId: string, userId: s
     }
 }
 
+export async function addUsersToTeam(serverUrl: string, teamId: string, userIds: string[]) {
+    let client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        EphemeralStore.startAddingToTeam(teamId);
+
+        const members = await client.addUsersToTeamGracefully(teamId, userIds);
```

Comment:
If the users are added to the team why are we not adding the relation to the database?

Reply:
Updated